src/dasi/api/Dasi.cc

```diff
@@ -123,27 +125,28 @@ fdb5::Config construct_config(const char* dasi_config, const char* application_c
 
 //----------------------------------------------------------------------------------------------------------------------
 
-/// @note We use an internal implementation for two reasons:
-///       (i) It ensures that the interface is kept as clean as possible
-///      (ii) It ensures that no FDB headers are exposed through DASI
-
 class DasiImpl {
 
 public: // methods
-
-    DasiImpl(const char* dasi_config, const char* application_config) :
-        mainHelper_(),
-        fdb_(construct_config(dasi_config, application_config)) {}
+    DasiImpl(const char* dasi_config, const char* application_config):
```

Comment:
I'm not convinced that the formatting changes are an improvement, just by increasing density.

And they make the changeset harder to read.